code/multitab-validation/SingleTabPage.php

```diff
@@ -16,6 +16,15 @@ class SingleTabPage extends Page
 {
     private static $table_name = 'SingleTabPage';
 
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+        // CWP\CWP\PageTypes\BasePage::getCMSFields() adds "Comments", "Tags", "Related Pages" tabs
+        // We should remove them to test that the validation works
+        $fields->removeByName(['Comments', 'Tags', 'RelatedPages']);
```

Comment:
`CWP\CWP\PageTypes\BasePage::getCMSFields()` adds "Comments", "Tags", "Related Pages" tabs. It means `SingleTabPage` has than a one tab.

Reply:
Please revert this change. We shouldn't be removing stuff that's added in the sink. The whole point of the sink is to check that things do work nicely together out of the box.